charts/application/values.yaml

```diff
@@ -21,6 +21,8 @@ autoscaling:
 
 # Check if the container is running
 livenessProbe:
+  # Disable the liveness probe
+  disabled: false
```

Comment:
I think because we already used the term `enabled` for other components, we should use it for these as well.

Reply:
That was my first thought as well.
The problem is that when I set `service.enable = true` as default, I would introduce a breaking change in the `k8-service`:

```
{{- if not (and .Values.canary.enabled .Values.istio.enabled ) }}
```

This does not work the same if I introduce the `service.enabled`

If you have a different solution, I'm open. My brain is fried anyways :D

Reply:
Used 2nd solution